qa/vagrant/src/test/resources/packaging/scripts/plugins.bash

```diff
@@ -36,6 +36,8 @@ install_plugin() {
 
     assert_file_exist "$ESPLUGINS/$name"
     assert_file_exist "$ESPLUGINS/$name/plugin-descriptor.properties"
+    #check we did not accidentially create a log file as root as /usr/share/elasticsearch
+    assert_file_not_exist "/usr/share/elasticsearch/logs"
```

Comment:
Is it worth adding an assertion that stdout doesn't contain any logging output by default?